scala/src/main/scala/com/quby/demo/databricks/util/SparkActiveSession.scala

```diff
@@ -3,6 +3,9 @@ package com.quby.demo.databricks.util
 import org.apache.spark.sql.SparkSession
 import org.slf4j.{Logger, LoggerFactory}
 
+/***
+  * Extend this trait for having quick access to current SparkSession and logger.
```

Comment:
Is this here so that current SparkSession always runs in utc timezone? Why does the trait need access to current sparksession and logger?

Reply:
The spark session needs to run on UTC timezone because this way the timestamp handling is always the same even on your local development machine.
The trait itself does not need access to the logger and spark session, but the classes that inherit it do.
We use this trait as shortcut to get those.